src/Controller.php

```diff
@@ -8,7 +8,6 @@
 use atk4\core\FactoryTrait;
 use atk4\core\InitializerTrait;
 use atk4\core\TrackableTrait;
-use Closure;
 use DateTime;
```

Comment:
same :)

Reply:
and in most cases you want to use \DateTimeImmutable for your data and \DateTimeInterface for datetime object detection ;)